Lib/extractor/formats/opentype.py

```diff
@@ -50,7 +50,7 @@ def extractFontFromOpenType(
         destination.kerning.update(kerning)
     for function in customFunctions:
         function(source, destination)
-    if doInstructions:
+    if doInstructions and "glyf" in source:
```

Comment:
can this if condition be handled in side `extractInstructions` like you did other if statements? 

see https://github.com/robotools/extractor/blob/master/Lib/extractor/formats/opentype.py#L423

Reply:
Sure! I moved the condition.